src/main/java/org/threadly/concurrent/AbstractPriorityScheduler.java

```diff
@@ -624,6 +635,18 @@ public long getRunTime() {
     }
     
     @Override
+    public long getScheduleDelay() {
```

Comment:
This actually was a fix...previously we were returning Clock.accurateForwardProgressingMillis() - Long.MAX_VALUE, which means our check in workerIdle was not even working (and thus could have been a bug in itself).